runtime/mainnet/src/config/proxy.rs

```diff
@@ -41,6 +39,17 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 	}
 }
 
+parameter_types! {
+	// One storage item; key size 32, value size 16.
+	pub const ProxyDepositBase: Balance = deposit(1, 48);
```

Comment:
we have 6 times the amount of bytes Polkadot has. Any reason not to just align to the Polkadot relay chain on this: https://github.com/polkadot-fellows/runtimes/blob/main/relay/polkadot/src/lib.rs#L977

Reply:
Certainly. Double checking after your comment I'm also seeing that I did the following miscalculation:

the number we have for `ProxyDepositBase` was the result of adding `AccountId` + `Balance`, but I realize that we should be accounting for 8 bytes, which is the correct length of the key of the map.

I also added the bytes for `Balance` because the deposited amount is always stored. Although, as it is the size corresponding to the deposit itself, I could see why it is not being included on Polkadot's configuration.

`ProxyDepositFactor` reasoning in the comment above its declaration, what's the concern ?

Reply:
[d4c936e](https://github.com/r0gue-io/pop-node/pull/441/commits/d4c936e5bca5659e6ed1324b4156ab6f82e1866e)

As discussed off GH this change updates ProxyDepositBase from  `deposit(1, 48)` to `deposit(1, 40)`.